### DISCUSSION TITLE
初次使用体验

### DISCUSSION THREAD

**Discussion by DryIce-cc** [Class-Widgets/Class-Widgets#79] [Q&A问答]
@RinLit-233-shiroko 开发者您好，看到您的 Bilibili 快闪视频了，您视频简介 Github 链接挂错了（）

应用主界面的样式外观很好看。这个半透明的悬浮效果很好看，配色、音效、每科的主题色晕染也很细节。♪(^∇^*)
设置界面在 Windows 10 上似乎没有窗口阴影，有点难调整窗口大小，希望尽量能修一下？
时间线编辑的逻辑作为新手稍稍有点难上手，不过探索了一下基本上也会用了（）
每种科目的对应图标都很精致欸，可以问问图标和音效资源是从哪里找到的吗，谢谢（ :wink:

祝愿 Class Widgets 越做越好。 :smiling_face_with_three_hearts:

**Comment by jizilin6732** [comment]
捉

**Comment by ZianTT** [comment]
> @DryIce-cc Discussion 板块已经开放，您不妨将使用体验及评价移动到 Discussion 板块。
> 感谢您的理解。

将Issue操作迁移至Discussion并删除原Issue需要操作人拥有Discussion及Issue管理权限，发布人无该权限操作。

**Comment by fred913** [comment]
@MKStoler4096 GitHub 有**直接将 Issue 迁移到 Discussion 的功能**，还能**保留所有人的回复**，这一操作需要**管理权限**。他**没说要删除**。
